src/App.scss

```diff
@@ -1,15 +1,8 @@
 .App {
-  display: flex;
   height: 100vh;
```

Comment:
```suggestion
```

Actually please just drop this line -- I've tested it out and it actually won't do anything visually, and just arbitrarily limits the size of this element while more content just overflows.

Scrolling works out of the box since the whole document is scrollable so we don't need any special handling here.

Reply:
Done